src/azure-cli/azure/cli/command_modules/containerapp/_clients.py

```diff
@@ -1017,6 +1017,12 @@ def stop_job(cls, cmd, resource_group_name, name, job_execution_name, job_execut
                 cls.api_version)
 
         r = send_raw_request(cmd.cli_ctx, "POST", request_url, body=json.dumps(job_execution_names))
+
+        # if stop is called for a single job execution and status code is 200, return generic response
+        if job_execution_name and r.status_code == 200:
+            return "Job Execution: "+job_execution_name +", stopped successfully."
```

Comment:
could you print the message in custom.py instead of client.

Reply:
@Juliehzl, moved the generic message to be returned from custom.py.